extras/signers.json

```diff
@@ -44,6 +44,21 @@
     "nanexcool": "0x823DF0278e4998cD0D06FB857fBD51e85b18A250",
     "davgarai": "0xAc1aA53108712d7f38093A67d380aD54B562a650"
   },
+  "arb_grant_treasury": {
+    "balancer": {
+      "tritium": "0xcf4fF1e03830D692F52EB094c52A5A6A2181Ab3F",
+      "mikeb": "0xc4591c41e01a7a654B5427f39Bbd1dEe5bD45D1D",
+      "xeonus": "0xb66941ffab0bad8f118c022921d8ac6f238c2eeb"
```

Comment:
I confirm I control this address. This address has been freshly set up for this purpose and is not associated with any other multi-sig and has no transaction history.